src/ex6/client/src/components/footer/footer.jsx

```diff
@@ -0,0 +1,35 @@
+
+import React from "react";
+import PropTypes from "prop-types";
+import "./footer.css";
+import ItemClient from "../../services/item_client.js";
+import { Button} from "monday-ui-react-core";
+
+const Footer = ({setTodoList,listTodo}) => {
+  const itemClient = new ItemClient();
+
+  const deleteListToDo = async () => {
+      await itemClient.deleteAllTodo();
+      setTodoList([]);
+  };

```

Comment:
the component has too much responsibility
- it makes the call to the server to delete the todos
- it cleans the todoList state in the parent

it would have have been better separated if this function was in the parent component